src/components/cylc/tree/GScanTreeItem.vue

```diff
@@ -33,6 +34,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       :to="workflowLink"
       :class="nodeClass"
       class="flex-grow-1 flex-shrink-1 px-2 ml-1"
+      :density="compactMode ? null : 'compact'"
```

Comment:
Wrong way round, also `null` doesn't work:
```suggestion
      :density="compactMode ? 'compact' : 'default'"
```
Or if we want to reduce the default padding slightly:

```suggestion
      :density="compactMode ? 'compact' : 'comfortable'"
```

Reply:
And yet it works?!

Tried out comfortable, that looks like a sensible default to me.

Reply:
(the original commit seems to have worked)

I took your suggestion.